Resources/Prototypes/_Impstation/Entities/Objects/Weapons/Guns/Shotguns/shotguns.yml

```diff
@@ -1,5 +1,5 @@
 - type: entity
-  name: Stun Enforcer
+  name: stun enforcer
```

Comment:
just "stun" being decapitalised is fine, i should have communicated this better. the guns in-game have capital letters, so "stun Enforcer" would be good